src/objectives/subsampled.jl

```diff
@@ -0,0 +1,100 @@
+
+"""
+    Subsampled(objective, batchsize, data)
+
+Subsample `objective` over the dataset represented by `data` with minibatches of size `batchsize`.
+
```

Comment:
Could you comment on what happens if `batchsize` does not divide `length(data)`, or whether that's significant at all?